app/models/concerns/hydranorth/generic_file/doi_states.rb

```diff
@@ -54,16 +56,23 @@ def withdraw_doi
         end
 
         def handle_doi_states
-          return unless doi_fields_present?
-          # TODO: handle unminted/unsynced states?
-          if doi.blank? # Never been minted before
-            created!(id) if !private? && unpublished?
-          else
-            # If private, we only care if visibility has been made public
-            # If public, we care if visibility changed to private or doi fields have been changed
-            if (unpublished? && transitioned_from_private?) || (available? && (doi_fields_changed? || transitioned_to_private?))
-              altered!(id)
+          if skip_handle_doi_states.blank?
```

Comment:
Maybe throw in a quick comment here (or on the attr above) just to explain the motivation behind this and the semantics of the flipflop